.github/actions_scripts/analyse_git_reference.py

```diff
@@ -64,7 +64,7 @@ def set_github_variable(varname, varval):
 release_version_name = get_release_version_name(jamulus_pro_version)
 
 fullref = os.environ['GITHUB_REF']
-publish_to_release = fullref.startswith('refs/tags/r')
+publish_to_release = bool(re.match(r'^refs/tags/r\d+_\d+_\d+\S*$', fullref))
```

Comment:
So we need to remember using _ instead of . everywhere on git.

(I'm not a fan of two different versions of describing the same version, but as far as I remember it is needed somehow).

Reply:
> So we need to remember using _ instead of . everywhere on git.

Yes.

> (I'm not a fan of two different versions of describing the same version, but as far as I remember it is needed somehow).

Agreed. I'd be more happy if we were following the more common vA.B.C scheme. If we decide to go that route we should do that properly and adjust all places though. :)
It's just that I think it will cause some breakage (e.g. people checking out git tags have to adjust, scripts may have to be updated, etc.).
If you want to make it happen I suggest opening an issue... :)